public/sandbox-data/apprenticeship-levy/employers/222hmrctest2/employed/AA234567C.json

```diff
@@ -0,0 +1,10 @@
+{
+  "status": 200,
+  "jsonBody": {
+    "empref": "222/hmrctest2",
+    "nino": "AA234567C",
```

Comment:
Use of Nino

Reply:
Ill address this in a later ticket. This is a bigger piece of work as App-levy is using itself as a stub. We need to create a separate microservice for this.